### PR TITLE
docs: update screenshots

### PR DESCRIPTION
**Pull request by rollerchloe**

### CODE REVIEW

File: docs/ethereum-section/issue-document.md [hunk 1]
```diff
@@ -23,10 +22,12 @@ You will be prompted for the password that you used while creating the wallet. Y
 ✔  success   Document/Document Batch with hash 0x80cc53b77c0539fc383f8d434ac5ffad281f3d64ae5a0e59e9f36f19548e1fff has been issued on 0xBBb55Bd1D709955241CAaCb327A765e2b6D69c8b
 ```
 
+>**Note:** The above command does not create any new file or folder. Instead, the documents issued to the document store are still in the `wrapped-documents` folder.
```
Comment (yapyuyou) [inline docs/ethereum-section/issue-document.md:25]:
```suggestion
>**Note:** The above command does not create any new files or folders. Once issued to the document store, the file(s) in the `wrapped-documents` folder will be verifiable.
```

**Review by yapyuyou — CHANGES_REQUESTED** [review]

File: docs/did-section/create.md [hunk 2]
```diff
@@ -35,8 +35,12 @@ Run the command:
    ```
 The response looks like the following:
    ```text
-   ⚠  warning   You are about to reveal the private key of your wallet. Please type the following word into the terminal to prove that you understand the risks: environmental-maroon-scorpion
-   ? ack: environmental-maroon-scorpion
+   ⚠  warning   You are about to reveal the private key of your wallet. Please type "yes" (without quotes) into the terminal to prove that you understand the risks
+   ? ack: (node:3437) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply docs/did-section/create.md:39]: updated to: 
`   ⚠  warning   You are about to reveal the private key of your wallet. Please type "yes" (without quotes) into the terminal to prove that you understand the risks`
   
`? ack: yes`

File: docs/distribute-section/oa-embed-html.md [hunk 3]
```diff
@@ -22,13 +22,17 @@ To continue, follow the steps below:
 
     You will use a [sample template](https://github.com/Open-Attestation/oa-embedded-html/blob/master/samples/healthcert-pdt-oa-embedded-uri-fragment-sample.html).
 
-3. Modify the button in the HTML file to contain the Verify URL. 
+3. Modify the **Proceed** button in the HTML file to contain the Verify URL. 
 
-    This button will redirect the user to the [Verify page](https://www.verify.gov.sg/) which will verify and render the OA document.
+4. Double-click the HTML file to preview it in a browser.
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply docs/distribute-section/oa-embed-html.md:27]: updated

File: docs/ethereum-section/issue-document.md (hunk 1 above)
Comment (rollerchloe) [reply docs/ethereum-section/issue-document.md:25]: updated

**Pull request merged**